docs/startup/preview/index.md

```diff
@@ -247,12 +246,14 @@ MessageStream<String> shortWords = words.filter(word -> word.size() < 3);
 
 ### PartitionBy
 Re-partitions this MessageStream using the key returned by the provided keyExtractor and returns the transformed MessageStream. Messages are sent through an intermediate stream during repartitioning.
```

Comment:
Since this is the first place we use operatorId, do you think it would be helpful to briefly explain what it's for?
Or at least point them to the javadoc for an explanation.

Reply:
Good catch. I've updated the docs to add info on operatorId. Thanks!